.env.example

```
@@ -1,2 +1,136 @@
+IMAGE_DIR=
+
+# Optional PARSEC hardware-backed identity.
+PARSEC_ENABLED=false
+PARSEC_SOCKET=/run/parsec/parsec.sock
+
+# Ground Control
+
+# Harbor source registry.
+HARBOR_URL=
+HARBOR_USERNAME=
+HARBOR_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HARBOR_PASSWORD key should go before the HARBOR_URL key

(UnorderedKey)